frontend/ui/widget/textboxwidget.lua

```diff
@@ -580,6 +831,17 @@ function TextBoxWidget:paintTo(bb, x, y)
     bb:blitFrom(self._bb, x, y, 0, 0, self.width, self._bb:getHeight())
 end
 
+function TextBoxWidget:onCloseWidget()
+    -- free when UIManager:close() was called
```

Comment:
Free when UIManager:close() was called.

(I mean, it just looks so out of place with the next comment. :-P)

Reply:
Yes :) That's verbatim from the other widgets that have it, so, I just kept it for coherency even if it sounds gramatically strange.
And... it's I that introduced that strange wording :)